core/src/cubos/core/io/glfw_window.cpp

```diff
@@ -23,12 +23,19 @@ static Key glfwToCubosKey(int key);
 
 #endif // WITH_GLFW
 
+#define UNSUPPORTED()                                                                                                  \
+    do                                                                                                                 \
+    {                                                                                                                  \
+        CUBOS_CRITICAL("Unsupported when building without GLFW");                                                      \
+        abort();                                                                                                       \
+    } while (0)
+
 GLFWWindow::GLFWWindow()
 {
 #ifdef WITH_GLFW
     if (!glfwInit())
     {
-        logCritical("OGLRenderDevice::OGLRenderDevice() failed: glfwInit() failed");
```

Comment:
Can be replaced with the new `UNSUPPORTED` macro

Reply:
The `UNSUPPORTED` macro is being used to log the error of trying to use GLFW when you built cubos without it.
You may have built it with GLFW and still glfwInit() fails, so its a different kind of error.

Reply:
Ah, alright then :+1: